### ISSUE TITLE
Connion to graph  is not working

### ISSUE THREAD

**Issue by shanvelc** [aws/graph-explorer#54]
I have followed the installation guide and was able to install the graph explorer. But unable to connect to graph DB. Could you please provide the details what i am missing.Thanks.

**Comment by triggan** [comment]
Can you please provide further details about your configuration? What graph database are you attempting to connect to?  Where are you deploying graph explorer? Etc.

**Comment by sanchit231** [comment]
I am also having connectivity issue. Deployed graph explorer on EC2 machine and getting synchronization issue with Neptune DB.

**Comment by shanvelc** [comment]
Now we are able to connect. But data is not getting viewed when we create a RDF manually. Instead if we use SPARQL and insert data it works. What would be the issue?

**Comment by shanvelc** [comment]
> Can you please provide further details about your configuration? What graph database are you attempting to connect to? Where are you deploying graph explorer? Etc.

Check if the port of 8182 is open in Neptune SG for EC2 proxy to access.

**Comment by triggan** [comment]
> Now we are able to connect. But data is not getting viewed when we create a RDF manually. Instead if we use SPARQL and insert data it works. What would be the issue?

Not sure I follow what you mean by "create a RDF manually".  Can you please elaborate?

In Neptune, Property Graph and RDF data are stored separately.  So if you write data using SPARQL/RDF, you can only access that data using SPARQL in RDF format.  Subsequently, if you write data using Gremlin or openCypher (both Property Graph languages) you can only then read that data using Gremlin or openCypher.

**Comment by sanchit231** [comment]
> > Can you please provide further details about your configuration? What graph database are you attempting to connect to? Where are you deploying graph explorer? Etc.
> 
> Check if the port of 8182 is open in Neptune SG for EC2 proxy to access.

Yes it is open when I try to telnet neptune from EC2 machine. Its working fine on 8182 port
